dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/EscapeHandlerInvocationHandler.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.common.jaxb;
+
+import java.io.Writer;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.cxf.common.logging.LogUtils;
+
+public final class EscapeHandlerInvocationHandler implements InvocationHandler {
+
+    private Object target; 
+    private static final Logger LOG = LogUtils.getLogger(EscapeHandlerInvocationHandler.class);  // Liberty Change 
+    public EscapeHandlerInvocationHandler(Object obj) {
+        target = obj;
+    }
+    @Override
+    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+        Object result = null;
+        if ("escape".equals(method.getName()) && args.length == 5) {
+            if ((Integer)args[1] == 0 && (Integer)args[2] == 0) {
+                Writer writer = (Writer)args[4];
+                writer.write("");
+	        if (LOG.isLoggable(Level.FINE)) {  // Liberty Change start
```

Comment:
This is purely formatting, but I think we need either commit to using start and stop blocks on new lines, or make the format for logging additions a one line comment. Its harder to read the change comments when its not on a new line. 
```suggestion
                // Liberty Change start
	        if (LOG.isLoggable(Level.FINE)) {  
	            LOG.fine("escape method returning null.");
	        } 
	        // Liberty Change end
```

or 
```

        if (isLoggableInfo) {  
	       LOG.info("Ignoring throwable: " + t); // Liberty Change 
         }
```

Reply:
yes, I agree, we need to be consistent, makes it easier to identify liberty changes